servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -185,19 +220,13 @@ def query_jobs(body):
         raise InternalServerError(response.json().get('message'))
     response.raise_for_status()
 
-    # Only list parent jobs
-    now = datetime.utcnow()
-    results = [
-        format_job(job, now) for job in response.json()['results']
-        if _is_parent_workflow(job)
-    ]
-    # Reverse so that newest jobs are listed first
-    results.reverse()
+    return response.json()['totalResultsCount']
```

Comment:
Since we can get `totalResultsCount` now, do you think we could replace the "many" on the page?
![screen shot 2018-02-06 at 4 17 25 pm](https://user-images.githubusercontent.com/14366016/35884620-433fa0a2-0b59-11e8-8de0-b308df9b3d20.png)

Reply:
This is still not possible for dsub, however it could potentially be somehow encoded in the `/capabilities` end point in the future.

Reply:
I see, thanks for the explanation here! @bfcrampton